CodeFormatter/formatoptions.h

```diff
@@ -112,6 +112,16 @@ enum PHPFixserFormatterStyle {
     kPcfConcatSpaceOne = (1 << 19),
     kPcfEmptyReturnStrip = (1 << 20),
     kPcfEmptyReturnKeep = (1 << 21),
+    kPcfCombineConsecutiveUnsets = (1 << 22),
+    kPcfLinebreakAfterOpeningTag = (1 << 23),
+    kPcfMbStrFunctions = (1 << 24),
+    kPcfNoBlankLinesBeforeNamespace = (1 << 25),
+    kPcfNoMultilineWhitespaceBeforeSemicolons = (1 << 26),
+    kPcfNoNullPropertyInitialization = (1 << 27),
+    kPcfNoPhp4Constructor = (1 << 28),
+    kPcfNoShortEchoTag = (1 << 29),
+    kPcfNoUnreachableDefaultArgumentValue = (1 << 30),
+    kPcfNoUselessElse = (1 << 31),
```

Comment:
Actually, when I think of it, we are using `C++11`, this means that you can use `class enum` like this:

```C++
enum class eEnum : unsigned long long {
    kOne = (1ULL << 0),
    kTwo = (1ULL << 1),
    kThree = (1ULL << 2),
    kLast = (1ULL << 33),
};
```
this allows you to set the type of the `enum` to `unsigned long long`
Note: whith class enum you need to reference it like this:
`eEnum::kOne`